### DISCUSSION TITLE
Feature Request: Push Notifications and Badges for Requested Reviews

### DISCUSSION THREAD

**Discussion by mbbischoff** [community/community#11] [Mobile] [answered]
I use the app [Quests](https://apps.apple.com/us/app/quests/id1447415753?mt=12) in my Mac menu bar to keep track of how many pull requests I need to review. It would be great if I could get a push notification for these on my iPad and optionally have the app badge reflect the number of reviews I need to take care of as well.

**Comment by candyho** [comment, marked as answer]
Hey @mattbischoff , thanks for sharing your feedback. Good news is that supporting push notifications for requested reviews is something on our radar - so please watch this space. Will pass the suggestion on badges along to the team!

**Reply by Billybryant756** [reply]
Get my  payments please

**Reply by farmtestnick** [reply]
> Hey @mattbischoff , thanks for sharing your feedback. Good news is that supporting push notifications for requested reviews is something on our radar - so please watch this space. Will pass the suggestion on badges along to the team!

Is there any update on if/when badges will be enabled? It would help our team too. Thanks!

**Comment by MartinNowak** [comment]
The GitHub app on Android seems to support this now. Might apply to iOS as well.
https://github.blog/2021-03-30-new-push-notifications-scheduling-releases-github-mobile/

**Reply by aconsuegra** [reply]
It's supported on iOS too!
